lib/src/constants/typedefs.dart

```diff
@@ -75,3 +78,8 @@ typedef AssetsChangeRefreshPredicate<Path> = bool Function(
   MethodCall call,
   Path? path,
 );
+
+typedef SpecialItemModel = ({
```

Comment:
If this was used internally only, move it to where the usages are.

Reply:
done moving to asset_picker_builder_delegate.dart file.